packages/react-await/src/awaitOpetions.spec.tsx

```diff
@@ -1,8 +1,9 @@
 import { FALLBACK, TEXT } from '@suspensive/test-utils'
```

Comment:
awaitOpetions.spec.tsx -> awaitOptions.spec.tsx

Reply:
I modified it https://github.com/toss/suspensive/commit/df5377a6e51eba15ec7a8dfb355ce7b043057dbf